### DISCUSSION TITLE
Insert row in roles table whenever we create a permission role in Hasura

### DISCUSSION THREAD

**Discussion by alveshelio** [nhost/nhost#31] [Ideas]
Hi guys,

I'm not sure if this is the right place for this discussion, if not I can create a new one in the appropriate project.

Right now whenever we create a permission role in Hasura we need to make sure we add the same role in the `"auth"."roles"` table. It would be nice if this would be done automatically. 
It should create a migration that would be applied.

**Comment by elitan** [comment]
The current disconnect between Nhost Auth and Hasura console is too big right now.

But I really like this idea. They should definitely be in sync I think. And I think we can make this happen in the future. 👍

**Reply by alveshelio** [reply]
With version 2 and above of Hasura which introduces a metadata api: https://hasura.io/docs/latest/graphql/core/api-reference/metadata-api/index.html#metadata-apis, this should be possible

**Reply by elitan** [reply]
I'm not sure I understand how v2 would help with this issue. Could you elaborate?

**Comment by MaxSchilling** [comment]
Would it be possible to add the active/inactive state(email Verified yes/no) in the users table as well? I‘m struggling to get the information if a user has verified his/her email. Or would that be an own issue?

**Reply by elitan** [reply]
You can check `auth.accounts.active` for that. Each user in the `users` table has an associated `account` in the `auth` schema. Does this help?

**Reply by MaxSchilling** [reply]
I guess! Will try, thanks
